tests/integration/test_responses_pages.py

```diff
@@ -36,7 +36,7 @@ def test_get_question_responses_page(django_app):
         assert f"{answers[i].free_text}" in page_content
         assert f"{answers[i].multiple_choice[0]}" in page_content
         if answers[i].free_text:
-            assert f"{answers[i].theme.summary}" in page_content
+            assert f"{answers[i].theme.short_description}" in page_content
```

Comment:
Is this now testing something different (summary -> summary, label -> short_description, but here summary -> short_description)